roles/redis_7_4/defaults/main.yml

```diff
@@ -29,7 +29,7 @@ redis_group: redis
 #
 
 REDIS_REPO: deb https://packages.redis.io/deb {{ ansible_distribution_release }} main
-REDIS_VERSION: 6:7.2.5-1rl1~jammy1
```

Comment:
Could you rename the role folder to redis_7? The role can install any 7.x redis version by changing the `REDIS_VERSION` version.